src/local-history.ts

```diff
@@ -9,8 +9,18 @@ export interface HistoryFileProperties {
     parentFileName: string;
 }
 
+export const maxEntriesPerFile = 'local-history.maxEntriesPerFile';
+
 export class LocalHistoryProvider {
-    private historyFiles: any[] = [];
+    fileLimit: number;
```

Comment:
Why name it `fileLimit` and not `maxEntriesPerFile` to align with the preference.